src/wp-admin/includes/class-wp-debug-data.php

```diff
@@ -1024,6 +868,162 @@ public static function debug_data() {
 		return $info;
 	}
 
+	public static function get_wp_core(): array {
```

Comment:
Mind adding the function header/docblock?

Reply:
Oh mea culpa.